spec.md

```diff
@@ -164,49 +202,100 @@ echo "SPDX-License-Identifier: MIT" > file.txt
 # REUSE-IgnoreEnd
 ```
 
-### In-line snippet comments
-
-If a copyright and/or licensing info is to apply only to a certain snippet
-instead of the whole file, SPDX snippet tags SHOULD be used (as defined in [SPDX
-Specification, Annex H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
-
-Such an annotated snippet block MUST start with `SPDX-SnippetBegin` to mark its
-beginning and end with `SPDX-SnippetEnd` to mark the snippet's end.
-
-Do note that SPDX snippet tags MUST start with `SPDX-Snippet`, meaning that the
-correct copyright notice in a snippet is `SPDX-SnippetCopyrightText`.
-
-Example:
-
-```
-# SPDX-SnippetBegin
-# SPDX-License-Identifier: MIT
-# SPDX-SnippetCopyrightText: 2022 Jane Doe <jane@example.com>
-
-{$snippet_code_goes_here}
-
-# SPDX-SnippetEnd
+### REUSE.toml
+
+Copyright and Licensing Information MAY be associated with a file through a
+`REUSE.toml` file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
+
+A `REUSE.toml` file MAY be located in any directory, and can cover files that are
+within its directory or deeper. You MAY have multiple `REUSE.toml` files in
+different directories.
```

Comment:
Shall we mention that the file is a TOML file, e.g. also with a link to https://toml.io/en?

It's obvious somehow, but if people ask about the syntax, commenting scheme or so...

Reply:
agree